src/pages/Home.js

```diff
@@ -1,22 +1,42 @@
 import { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import TutoriesGallery from '../components/home/TutoriesGallery';
+import { useCurrentUserQuery } from '../api/usersData';
+import Login from './Login';
 import {
-  translateLeft, translateRight, updateHasReachedMaxScrolled, updateIsComputerWidth,
+  translateLeft, translateRight, updateHasReachedMaxScrolled, updateIsComputerWidth, updateUser,
 } from '../redux/tutories/tutoriesSlice';
+
 /* TODO: tutories should be fetched from API */
 
 export default function Home() {
   const dispatch = useDispatch();
+  // const [user, setUser] = useState({
+  //   loggedIn: false,
+  //   user: {},
+  // });
```

Comment:
I kindly suggest you remove the commented code since it has no use case in the repository or deployed project.